.circleci/config.yml.j2

```diff
@@ -43,10 +43,12 @@ jobs:
 
   unittest-charts:
     docker:
-      - image: quay.io/astronomer/ci-helm-release:2022-04
+      - image: quay.io/astronomer/ci-helm-release:2022-03
```

Comment:
Why was this reverted to an old version? It should stay the same or go forward if possible.

Reply:
Aah, I missed it again. Updated to newer one.